build/tutorials/tutorials-contest/avalanche-erc721-tutorial/README.md

```diff
@@ -0,0 +1,1170 @@
+* The [dotenv](https://www.npmjs.com/package/dotenv) is a zero-dependency module that loads environment variables from a .env file into process.env. We do not want to leak our mnemonic to other people after all;
+* The [@openzeppelin/test-helpers](https://docs.openzeppelin.com/test-helpers/0.5/) is a library that will helps us test when transactions revert and also handle Big Numbers for us. It is a dev dependency;
+* The [solidity-coverage](https://www.npmjs.com/package/solidity-coverage) is a library that we will use to check how much coverage our tests have. It is again a dev dependency;
+
+3. Now that we have all the necessary dependencies installed, let us go to the **truffle-config.js** in the root of our project and paste the following lines of code in there:
+
+```javascript
+const HDWalletProvider = require('@truffle/hdwallet-provider')
+require('dotenv').config()
+module.exports = {
+  networks: {
+    fuji: {
+      provider: () => {
+        return new HDWalletProvider({
+          mnemonic: process.env.MNEMONIC,
+          providerOrUrl: `https://avalanche--fuji--rpc.datahub.figment.io/apikey/${process.env.APIKEY}/ext/bc/C/rpc`,
+          chainId: '43113'
+        })
+      },
+      network_id: "*",
+      gasPrice: 225000000000
+    },
+    mainnet: {
+      provider: () => {
+        return new HDWalletProvider({
+          mnemonic: process.env.MNEMONIC,
+          providerOrUrl: `https://api.avax.network/ext/bc/C/rpc`,
+          chainId: '43114',
+        })
+      },
+      network_id: "*",
+      gasPrice: 225000000000
+    }
+  },
+  compilers: {
+    solc: {
+      version: "0.8.6"
+    }
+  },
+  plugins: ["solidity-coverage"]
+};
+```
+
+This file is the entrypoint of our Truffle project. As you can see, we specify two networks on which we would like to deploy our smart contracts after we are done with them, namely *fuji* and *mainnet*. We utilize the [@truffle/hdwallet-provider](https://www.npmjs.com/package/@truffle/hdwallet-provider) library, so that we provide a RPC to which we can connect to to deploy our contracts as well as a mnemonic which will be used to sign the transaction to do that. As you can see, some of the variables are accessed via process.env. These are defined in a separate **.env** file in the root of our project and have the following structure:
+```
+MNEMONIC='paste your metamask mnemonic here which is twelve words long believe me'
+APIKEY=YOUR_DATAHUB_API_KEY_FOR_THE_FUJI_TESTNET
+```
+Note: For the Fuji testnet I used [DataHub](https://datahub.figment.io/services/avalanche)'s testnet RPC. There is a free plan which you can use. For that you would need to register, grab your APIKEY and paste it into your **.env** file. 
+
+### Writing our first ERC721 contract
+
+1. Inside the **contracts/** folder of your Truffle project create a new Collectible.sol file and paste in the following code:
+
+```javascript
```

Comment:
Could you please use the correct code typing, so `solidity` here, instead of `javascript`. Check in other places, too.